kahuna/public/js/leases/leases.js

```diff
@@ -71,21 +75,30 @@ leases.controller('LeasesCtrl', [
                     ctrl.newLease.notes = noteWithClause.join(', ');
                 }
 
-                let syndLeases = ctrl.leases.leases.filter((l) =>
+                const incomingLeaseIsSyndication = ctrl.access.endsWith('-syndication');
+                const syndLeases = ctrl.leases.leases.filter((l) =>
                     l.access.endsWith('-syndication')
                 );
 
-                if (ctrl.access.endsWith('-syndication') && syndLeases.length > 0) {
-                        alertFailed('You can only set one syndication lease.');
-                } else {
-                    leaseService.batchAdd(ctrl.newLease, ctrl.leases, ctrl.images)
-                        .catch(() =>
-                            alertFailed('Something went wrong when saving, please try again.')
-                    )
-                    .finally(() => {
-                        ctrl.resetLeaseForm();
-                    });
+                if (incomingLeaseIsSyndication && syndLeases.length > 0) {
+                  const confirmText = ctrl.images.size > 1
+                    ? "One or more of the selected images have syndication leases. " +
+                      "These will be overwritten. Do you wish to proceed?"
+                    : "This image already has syndication rights. They will be overwritten. " +
```

Comment:
Slight wording change here: `This image already has syndication rights.` -> `This image already has a syndication lease.`

Reply:
👍